server/util/csv_valid_parse.js

```diff
@@ -0,0 +1,57 @@
+import parse from 'csv-parse/lib/sync';
+
+// promisified helper function, example usage:
+// parseCSVThenValidateHeaders()
+//  .then(objsArray => console.log(objsArray))
+//  .catch(err => console.log(err));
+
+export function extraHeadersError(headers) {
```

Comment:
Ah I see we use it during testing? Not super ideal, but I think we can get away with it for now. Ideally the test constructs its Error independent of the code that is under test.